target/linux/ramips/image/mt7621.mk

```diff
@@ -222,6 +222,15 @@ define Device/buffalo_wsr-600dhp
 endef
 TARGET_DEVICES += buffalo_wsr-600dhp
 
+define Device/cudy_wr1300
+  DTS := WR1300
```

Comment:
This is obviously not needed, as you see by the changed name for the dts file compared to where you copied this.

Reply:
Whoops. Removed.